src/test/java/com/wikia/webdriver/testcases/fandom/LoadMoreTest.java

```diff
@@ -0,0 +1,21 @@
+package com.wikia.webdriver.testcases.fandom;
+
+/**
+ * Created by psko on 8/15/16.
+ */
+
+import com.wikia.webdriver.common.templates.fandom.FandomTestTemplate;
+import com.wikia.webdriver.elements.fandom.components.LoadMore;
+import com.wikia.webdriver.elements.fandom.pages.MainPage;
+import org.testng.annotations.Test;
+
+@Test(groups = {"Fandom", "Fandom_Load_More"})
+public class LoadMoreTest extends FandomTestTemplate {
+
+    @Test
+    public void loadMoreButtonFunctionsCorrectly() {
+        MainPage mainPage = new MainPage().open();
+        LoadMore loadMore = new LoadMore().loadMoreBtn();
+        LoadMore topStories = new LoadMore().moreTopStories();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "topStories" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1481)